### PR TITLE
Support Ruby 2.2.x

### PR DESCRIPTION
**Pull request by morygonzalez**
- Drop unsupported Ruby version from .travis.yml
- Add support for Ruby 2.2.x
- Fix failing CI on travis-ci

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,11 +7,11 @@ gem 'sinatra-flash', '~> 0.3.0'
 gem 'i18n', '0.6.0'
 gem 'sinatra', '~> 1.4.2'
 gem 'sinatra-contrib', '~> 1.4.0'
-gem 'dm-core',          :git => 'git://github.com/datamapper/dm-core.git', :branch => 'release-1.2'
```
Comment (morygonzalez) [inline Gemfile]: dm-core's release-1.2 branch may not be exist any more.

File: Gemfile [hunk 2]
```diff
@@ -7,11 +7,11 @@ gem 'sinatra-flash', '~> 0.3.0'
...
+gem 'dm-core',          '1.2.1'
 gem 'dm-migrations',    '1.2.0'
 gem 'dm-timestamps',    '1.2.0'
 gem 'dm-validations',   '1.2.0'
-gem 'dm-types',         '1.2.0'
+gem 'dm-types',         '1.2.2'
```
Comment (morygonzalez) [inline Gemfile]: We need to update json in order to support Ruby 2.2. But dm-types@1.2.0's dependency was `json (~> 1.5.4)`. So I also update dm-types to 1.2.2 which supports newer version of json.

File: .travis.yml [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
-  - 1.9.2
-  - 1.9.3
   - 2.0
-  - 2.1.0
+  - 2.1.7
+  - 2.2.3
+addons:
+  apt:
+    packages:
+      - postgresql-server-dev-9.3
```
Comment (morygonzalez) [inline .travis.yml]: As travis migrated to container based environment, postgresql-server-dev package is needed to be installed to install do_postgres.

**Comment by morygonzalez** [comment]
ALL GREEEEEEN!!!

File: Gemfile [hunk 4]
```diff
@@ -4,20 +4,20 @@ gem 'bundler'
 gem 'rack'
 gem 'rack-flash', '~> 0.1.2'
 gem 'sinatra-flash', '~> 0.3.0'
-gem 'i18n', '0.6.0'
+gem 'i18n', '~> 0.6.0'
 gem 'sinatra', '~> 1.4.2'
 gem 'sinatra-contrib', '~> 1.4.0'
-gem 'dm-core',          :git => 'git://github.com/datamapper/dm-core.git', :branch => 'release-1.2'
+gem 'dm-core',          '1.2.1'
```
Comment (komagata) [inline Gemfile]: 💋

**Comment by komagata** [comment]
@morygonzalez thx!

**Pull request merged**